apps/angular/src/app/features/anime/pages/anime/anime-page.component.html

```diff
@@ -0,0 +1,45 @@
+@if (isLoading$ | async; as isLoading) {
+	<mat-progress-spinner color="primary" mode="indeterminate" class="spinner" diameter="36"></mat-progress-spinner>
+}
+
+@if (animes$ | async; as animes) {
+	<div class="table-wrapper">
+		<table mat-table [dataSource]="animes.items" class="mat-elevation-z8">
+
+			<ng-container matColumnDef="image">
+				<th mat-header-cell *matHeaderCellDef></th>
+				<td mat-cell *matCellDef="let anime">
+					<img src={{anime.image}} alt="{{anime.name}}" class="poster" />
```

Comment:
Is this a poster or a image? These are 2 different terms, I suggest to avoid confusion to use 1 of them everywhere, for Anime I like the term "Poster" better